### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by riddletd**
## Summary

The point of this file change is to improve readablilty and clarity for more efficient comprehension.

### CODE REVIEW

File: docs/core/testing/index.md [hunk 1]
```diff
@@ -1,32 +1,42 @@
...
+* [Visual Basic](../../visual-basic/index.md) 
 
-Unit test projects for .NET Core are supported for [C#](../../csharp/index.md), [F#](../../fsharp/index.md) and [Visual Basic](../../visual-basic/index.md). You can also choose between [xUnit](https://xunit.github.io), [NUnit](https://nunit.org) and [MSTest](https://github.com/Microsoft/vstest-docs).
+You can also choose between:
+* [xUnit](https://xunit.github.io) 
+* [NUnit](https://nunit.org)
+* [MSTest](https://github.com/Microsoft/vstest-docs)
 
-You can read about those combinations in these walkthroughs:
+Walkthroughs for combinations of the above:
```
Comment (BillWagner) [inline docs/core/testing/index.md]: This is a good change. This might be better: "You can learn more in the following walkthroughs:"

File: docs/core/testing/index.md [hunk 2]
```diff
@@ -38,9 +48,8 @@ You can read about those combinations in these walkthroughs:
 * Create unit tests using [*NUnit* and *Visual Basic* with the .NET Core CLI](unit-testing-visual-basic-with-nunit.md).
 * Create unit tests using [*MSTest* and *Visual Basic* with the .NET Core CLI](unit-testing-visual-basic-with-mstest.md).
 
-You can choose different languages for your class libraries and your unit test libraries. You can learn how by mixing and matching the walkthroughs referenced above.
+Further readings related to testing:
```
Comment (BillWagner) [inline docs/core/testing/index.md]: I'd rephrase this as "You can learn more in the following articles."

**Review by BillWagner — CHANGES_REQUESTED** [review]
Thanks for submitting these changes @riddletd We appreciate it.

There are a lot of improvements in this article. I had a few additional comments for you to address before we merge this change.

File: docs/core/testing/index.md [hunk 3]
```diff
@@ -1,32 +1,42 @@
 ---
 title: Unit Testing in .NET Core
-description: Unit testing has never been easier. See how to use unit testing in .NET Core and .NET Standard projects.
-author: ardalis
+description: Unit testing in .NET Core and .NET Standard projects.
+author: ardalis & riddletd
```
[the remark this answers is not in the document]
Comment (riddletd) [reply docs/core/testing/index.md]: I was wondering if I needed to add my ID when I made these changes, and this answers my question.  I will definitely remove it. Thank you.

File: docs/core/testing/index.md [hunk 4]
```diff
@@ -1,32 +1,42 @@
...
 ms.author: wiwagn
 ms.date: 08/30/2017
 ---
 # Unit testing in .NET Core and .NET Standard
 
-.NET Core has been designed with testability in mind, so that creating unit tests for your applications is easier than ever before. This article briefly introduces unit tests (and how they differ from other kinds of tests). Linked resources demonstrate how to add a test project to your solution and then run unit tests using either the command line or Visual Studio.
+.NET Core makes it easy to create unit tests. This article introduces unit tests and illustrates how they differ from other kinds of tests. The linked resources near the bottom of the page shows you how to add a test project to your solution. After you set up your test project, you will be able to run your unit tests through the command line or Visual Studio.
```
[the remark this answers is not in the document]
Comment (riddletd) [reply docs/core/testing/index.md]: Good eye. Thank you.

File: docs/core/testing/index.md [hunk 5]
```diff
@@ -1,32 +1,42 @@
...
 
-.NET Core 2.0 supports [.NET Standard 2.0](../../standard/net-standard.md). The libraries used to demonstrate unit testing in this section rely on .NET Standard and will work in other project types as well.
+.NET Core 2.0 supports [.NET Standard 2.0](../../standard/net-standard.md), and we will use it's libraries to demonstrate unit tests.
 
-Beginning with .NET Core 2.0, there are unit test project templates for C#, F# and Visual Basic.
+You are able to use built-in .NET Core 2.0 unit test project templates for C#, F# and Visual Basic as a starting point for your personal project.
```
[the remark this answers is not in the document]
Comment (riddletd) [reply docs/core/testing/index.md]: Good idea.

File: docs/core/testing/index.md [hunk 6]
```diff
@@ -1,32 +1,42 @@
...
+Having automated tests is a great way to ensure a software application does what its authors intend it to do. There are multiple types of tests for software applications. These include integration tests, web tests, load tests, and others. Unit tests test individual software components and methods. Unit tests should only test code within the developer’s control. They should not test infrastructure concerns. Infrastructure concerns include databases, file systems, and network resources.
```
[the remark this answers is not in the document]
Comment (riddletd) [reply docs/core/testing/index.md]: Good idea.

File: docs/core/testing/index.md (hunk 1 above)
Comment (riddletd) [reply docs/core/testing/index.md]: Thank you. I think you are right. That is a lot better.

File: docs/core/testing/index.md (hunk 2 above)
Comment (riddletd) [reply docs/core/testing/index.md]: I agree.

**Comment by BillWagner** [comment]
Thanks @mairaw I'd missed the update.

**Review by BillWagner — APPROVED** [review]

**Comment by BillWagner** [comment]
Thanks for making those additional updates @riddletd 
I'll :shipit: this PR now. You should see the updates on the live site in the next day or two, on our regular publishing cycle.

Thanks again for contributing. 🎆

**Comment by riddletd** [comment]
Thanks Bill! I'm excited I was able to contribute!
